src/reducer.ts

```diff
@@ -207,7 +211,8 @@ const ACTION_ORIGIN = Symbol('Action origin');
 export const combineReducers = <State>(
   seed: State,
   reducers: RegisteredReducer<State, any>[],
-  errorSubject: Subject<any> = defaultErrorSubject
+  errorSubject: Subject<any> = defaultErrorSubject,
+  performanceMarkerName?: string
```

Comment:
I "lazily" added this as a last optional argument here, but I'm struggling to see a better place to put it. Any thoughts on this @tlaundal ?

Reply:
Hmm, I think there isn't an obvious correct order here. Neither of them is more "important", and there is no guarantee that one of them will only be used together with the other.

Though, I hope this operator will turn more and more into an internal thing, so maybe the signature won't really matter that much?

The two alternatives I can think of:
 - A configuration object which takes either or both of them
 - Overloads and dynamic argument recognition, so the order doesn't matter. This should be possible as `Subject`/`object` and `string` are separate types

Of the two, I think the first one is the best, but I would also be fine with keeping it as it is.

Reply:
thanks for the input 👍 

I agree and will refactor to a config object